packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -58,11 +55,35 @@ import {
   getMediaType,
   getFullMimeWhitelist,
 } from '../message-utils'
-
 import { normalizeWebchatSettings } from '../components/webchat-settings'
+import { getProperty } from '../util/objects'
 
-import merge from 'lodash.merge'
-import { useAsyncEffect } from 'use-async-effect'
+export const getParsedAction = botonicAction => {
+  const splittedAction = botonicAction.split('create_case:')
+  if (splittedAction.length <= 1) return undefined
+  return JSON.parse(splittedAction[1])
+}
+
+/*
```

Comment:
/* is not jsdoc https://jsdoc.app/about-getting-started.html
Start comments with Capital